### PR TITLE
Add get file api

### PR DESCRIPTION
**Pull request by martyk13**
We still need to add the fetch in the git scanner but this should fix the scheduling issue at build time.

### CODE REVIEW

File: analysis-platform/src/main/java/com/kenesys/analysisplatform/api/TemplatesApi.java [hunk 1]
```diff
@@ -3,7 +3,14 @@
 import com.kenesys.analysisplatform.dao.TemplateDao;
```
Comment (linge365) [inline analysis-platform/src/main/java/com/kenesys/analysisplatform/api/TemplatesApi.java]: General : need to rename package in line with last week's discussions

Comment (martyk13) [reply analysis-platform/src/main/java/com/kenesys/analysisplatform/api/TemplatesApi.java]: yip!

Comment (martyk13) [reply analysis-platform/src/main/java/com/kenesys/analysisplatform/api/TemplatesApi.java]: I'm thinking for the package names we have:
com.amplify.ap
and
com.amplify.apc

and then make the project names 
analysis-platform
and
analysis-platform-controller

I'll have a play but pretty sure the project name doesn't need to be the final part of the package name (that was just part of the spring initalizer)

Comment (linge365) [reply analysis-platform/src/main/java/com/kenesys/analysisplatform/api/TemplatesApi.java]: try it - see how they look + not too cumbersome

**Pull request merged**